src/lib/integration_test_local_engine/mina_automation.ml

```diff
@@ -0,0 +1,462 @@
+open Core
+open Async
+open Currency
+open Signature_lib
+open Mina_base
+open Integration_test_lib
+
+let version = "3.9"
```

Comment:
Can we call this `docker_version` so it's more obvious what this string is versioning?